CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- New "tags" parameter to operations object in Swagger format.
 - Add changelog and Makefile.
 
-[Unreleased]: https://github.com/brainn-co/xcribe/compa...master
+[Unreleased]: https://github.com/brainn-co/xcribe/compa...master
```

Comment:
@RafaelCamarda This url is really weird. Wouldn't be compare here?

Reply:
Yes, thx @thiamsantos . The correct URL is
`https://github.com/brainn-co/xcribe/compare/0.3.0...master`

My bad in the first PR. This will be updated as well when running the release task.